tests/details.conf

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
How about merging this file with the environment.conf? Are they used for different kinds of tests?